src/bin/test_userproc_even.c

```diff
@@ -0,0 +1,29 @@
+int main();
+int _start() {
+    return main();
+
+    // invoke system call that kills the process
+}
+
+#include "../syscall.h"
```

Comment:
The Makefile is built around the executables being in bin/; we'd have to rework that if we want to get rid of the relative path. The Makefile also automatically moves compiled files in bin/ to the files.iso. I kind of like having them in their own folder to emphasize that they have different compiler rules and that they get copied to the ISO